Sources/HaCWebsiteLib/Controllers/EventBetaController.swift

```diff
@@ -0,0 +1,23 @@
+import Kitura
+import Foundation
+import HaCTML
+
+func getDate() -> Date {
+  let formatter = DateFormatter()
+  formatter.dateFormat = "yyyy/MM/dd HH:mm"
+  return formatter.date(from: "2017/12/13 14:31") ?? Date()
+}
+
+struct EventBetaController {
```

Comment:
Did this end up being useful for testing or just an extra hassle?

Reply:
It was useful for testing whether the "Today" applied for specific dates and times. Should I remove this from the PR before it's merged?

Reply:
Perhaps let's take this out for production yeah. It's worthwhile playing with these sandbox things though, I think they could be a lot more useful in the future.